### PR TITLE
add request logs for iOS:APNs and Android:FCM

### PR DESCRIPTION
**Pull request by keremoge**
APNs and FCM Api request logs added to see which requests are going to APNs And FCM servers

Example:

- [x] request going gorush:

```
{
  "notifications": [
    {
      "platform": 1,
      "priority": "high",
      "tokens": [
        "**********4wAABp_BOYhKWWMLfSF9GV52UO_rYvBoKbQuuuI6HWSfLdG7_t-ZhA2_Zg47BpQbzBegmHUxovPTp3rU0xajPAfnz9IkOb**********"
      ],
      "content_available": true,
      "category": "CONFIRM",
      "sound": {
        "critical": 1,
        "name": "default",
        "volume": 1
      },
      "alert": {
        "body": "iOS UAT başarılı bildirim geldi"
      },
      "data": {
        "customerNo": 12345,
        "languageCode": "tr",
        "specificCharText": "denemee"
      }
    }
  ]
}
```

- [ ] request going APNS: (**We need this**)

```
{
  "aps": {
    "alert": {
      "body": "iOS UAT başarılı bildirim geldi"
    },
    "category": "CONFIRM",
    "content-available": 1,
    "sound": {
      "critical": 1,
      "name": "default",
      "volume": 1
    }
  },
  "customerNo": 12345,
  "languageCode": "tr",
  "specificCharText": "denemee"
}
```

### CODE REVIEW

File: storage/redis/redis.go [hunk 1]
```diff
@@ -48,11 +48,9 @@ func (s *Storage) Init() error {
 		})
 	}
 
-	if err := s.client.Ping(s.ctx).Err(); err != nil {
-		return err
-	}
+	err := s.client.Ping(s.ctx).Err()
```
Comment (slimus) [inline storage/redis/redis.go:51]:
```suggestion
return s.client.Ping(s.ctx).Err()
```

File: notify/notification_apns.go [hunk 2]
```diff
@@ -406,7 +414,7 @@ func PushToIOS(req *PushNotification, cfg *config.ConfYaml) (resp *ResponsePush,
 Retry:
 	var newTokens []string
 
-	notification := GetIOSNotification(req)
+	notification, _ := GetIOSNotification(req)
```
Comment (slimus) [inline notify/notification_apns.go:417]: You can't ignore `err` here. In case of `err != nil`, `notification = nil` and we have panic later.

File: notify/notification_apns_test.go [hunk 3]
```diff
@@ -83,7 +83,7 @@ func TestIOSNotificationStructure(t *testing.T) {
 		URLArgs:  []string{"a", "b"},
 	}
 
-	notification := GetIOSNotification(req)
+	notification, _ := GetIOSNotification(req)
```
Comment (slimus) [inline notify/notification_apns_test.go:86]: please, add new tests for `err != nil` case

**Review by slimus — CHANGES_REQUESTED** [review]

File: storage/redis/redis.go (hunk 1 above)
Comment (keremoge) [reply storage/redis/redis.go:51]: thanks @slimus , I forgot to return 👍

File: notify/notification_apns_test.go (hunk 3 above)
Comment (keremoge) [reply notify/notification_apns_test.go:86]: I think just because failing to marshal the json shouldn't block the notification, I will modify the usages and only log as error.

File: notify/notification_apns.go (hunk 2 above)
Comment (keremoge) [reply notify/notification_apns.go]: Hi @slimus , just because **log** parser error, we don't have to return err parameter, so I removed it from return statement

File: notify/notification_apns_test.go (hunk 3 above)
Comment (keremoge) [reply notify/notification_apns_test.go]: As I mentioned before; 
just because **log** parser error, we don't have to return err parameter, so I removed it from return statement

**Review by keremoge — COMMENTED** [review]
seems good

**Comment by keremoge** [comment]
@appleboy can you approve this? It is helpful for debugging

**Comment by Spinaker70** [comment]
Indeed, the ability to see the contents of the request in the output is really missing...
